DATASETS.md

```diff
@@ -27,8 +27,9 @@ These datasets can be downloaded follow the instructions [here](https://github.c
 | [Embodied Question Answering](https://embodiedqa.org/) | MatterPort3D | [eqa_mp3d_v1.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/eqa/mp3d/v1/eqa_mp3d_v1.zip) | `data/datasets/eqa/mp3d/v1/` | [`datasets/eqa/mp3d.yaml`](habitat-lab/habitat/config/habitat/dataset/eqa/mp3d.yaml)                                               | 44 MB |
 | [Visual Language Navigation](https://bringmeaspoon.org/) | MatterPort3D | [vln_r2r_mp3d_v1.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/vln/mp3d/r2r/v1/vln_r2r_mp3d_v1.zip) | `data/datasets/vln/mp3d/r2r/v1` | [`datasets/vln/mp3d_r2r.yaml`](habitat-lab/habitat/config/habitat/dataset/vln/mp3d_r2r.yaml)                                       | 2.7 MB |
-| 🆕[Instance image goal navigation](https://arxiv.org/abs/2211.15876) | HM3D <br> (+Sem v0.1) | [instance_imagenav_hm3d_v1.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/imagenav/hm3d/v1/instance_imagenav_hm3d_v1.zip) | `data/datasets/instance_imagenav/hm3d/v1/` |  [`datasets/instance_imagenav/hm3d_v1.yaml`](habitat-lab/habitat/config/benchmark/nav/instance_imagenav/instance_imagenav_hm3d_v1.yaml) | 303 MB |
-| 🆕[Instance image goal navigation](https://arxiv.org/abs/2211.15876) | HM3D <br> (+Sem v0.2) | [instance_imagenav_hm3d_v2.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/imagenav/hm3d/v2/instance_imagenav_hm3d_v2.zip) | `data/datasets/instance_imagenav/hm3d/v2/` |  [`datasets/instance_imagenav/hm3d_v2.yaml`](habitat-lab/habitat/config/benchmark/nav/instance_imagenav/instance_imagenav_hm3d_v2.yaml) | 518 MB |
+| [Instance image goal navigation](https://arxiv.org/abs/2211.15876) | HM3DSem-v0.1 | [instance_imagenav_hm3d_v1.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/imagenav/hm3d/v1/instance_imagenav_hm3d_v1.zip) | `data/datasets/instance_imagenav/hm3d/v1/` |  [`datasets/instance_imagenav/hm3d_v1.yaml`](habitat-lab/habitat/config/habitat/dataset/instance_imagenav/hm3d_v1.yaml) | 303 MB |
+| [Instance image goal navigation](https://arxiv.org/abs/2211.15876) | HM3DSem-v0.2 | [instance_imagenav_hm3d_v2.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/imagenav/hm3d/v2/instance_imagenav_hm3d_v2.zip) | `data/datasets/instance_imagenav/hm3d/v2/` |  [`datasets/instance_imagenav/hm3d_v2.yaml`](habitat-lab/habitat/config/habitat/dataset/instance_imagenav/hm3d_v2.yaml) | 518 MB |
+| 🆕 [Instance image goal navigation](https://arxiv.org/abs/2211.15876) | HM3DSem-v0.2 | [instance_imagenav_hm3d_v3.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/imagenav/hm3d/v3/instance_imagenav_hm3d_v3.zip) | `data/datasets/instance_imagenav/hm3d/v3/` |  [`datasets/instance_imagenav/hm3d_v3.yaml`](habitat-lab/habitat/config/habitat/dataset/instance_imagenav/hm3d_v3.yaml) | 517 MB |
```

Comment:
```suggestion
| 🆕 [Instance image goal navigation](https://arxiv.org/abs/2211.15876) | HM3DSem-v0.3 | [instance_imagenav_hm3d_v3.zip](https://dl.fbaipublicfiles.com/habitat/data/datasets/imagenav/hm3d/v3/instance_imagenav_hm3d_v3.zip) | `data/datasets/instance_imagenav/hm3d/v3/` |  [`datasets/instance_imagenav/hm3d_v3.yaml`](habitat-lab/habitat/config/habitat/dataset/instance_imagenav/hm3d_v3.yaml) | 517 MB |
```

Reply:
That column is referencing the scene dataset, which is still `HM3DSem-v0.2` for the v3 IIN episode dataset.

Reply:
Oh okay got it